.idea/misc.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="TaskProjectConfiguration">
-    <server type="GitHub" url="https://github.com" />
-  </component>
-</project>
```

Comment:
This is not connected to RLD-44 but removed the GitHub server from WebStorm Tasks

Reply:
Awesome! Wanted to do that myself for a while now. Thanks!